addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -468,7 +468,16 @@ var FieldDate = InputField.extend({
      * @private
      */
     _makeDatePicker: function () {
-        return new datepicker.DateWidget(this, {defaultDate: this.value});
+        var datepicker_options;
+        try {
+            datepicker_options = this.attrs.options.datepicker;
```

Comment:
Isn't it simpler to do datepicker_options = this.attrs.options.datepicker || {} ?

Reply:
Hmm it must be that.

Reply:
tde is right

Reply:
I was worried about `this.attrs.options` being `undefined`, but I didn't know about `this.nodeOptions`. It's changed now.